src/components/form/form.jsx

```diff
@@ -128,3 +132,74 @@ Form.Feedback = ({ className, message }) => {
     </>
   );
 };
+
+Form.Combobox = ({ id, labelText, className, list, name, nameKey, idKey }) => {
+  const [selectedItems, setSelectedItems] = useState([]);
+  const [query, setQuery] = useState("");
+  const filteredList =
+    query === ""
+      ? list
+      : list.filter((item) => {
+          return item[nameKey].toLowerCase().includes(query.toLowerCase());
+        });
+  return (
+    <div className={`flex font-bold font-arial flex-col py-3 ${className}`}>
+      <label htmlFor={id} className={`mb-1.5 text-lg text-start `}>
```

Comment:
The label has an htmlFor declared but it's never used to tie it to the combobox.input

Reply:
everything should be fixed now